tests/integration/test_outcome.py

```diff
@@ -1001,42 +1001,27 @@ def test_outcomes_do_not_aggregate(
     outcomes_consumer = outcomes_consumer(timeout=1.2)
 
     # Send empty body twice
-    event_id1 = _send_event(relay)
-    event_id2 = _send_event(relay)
+    _send_event(relay)
```

Comment:
The test is called `test_outcomes_do_not_aggregate`, maybe we should just delete it if it's no longer useful or at least rename it.

Reply:
👍 updated the test name and comments now.